docker-bake.hcl

```
@@ -22,6 +22,9 @@ variable "GITHUB_REF" {
   default = ""
 }
 
+variable "IMAGE_LOCAL" {
+}
```

I think we need a default like:

```suggestion
variable "IMAGE_LOCAL" {
  default = "buildkit-syft-scanner:local"
}
```